tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientSideEncryptionSmokeProseTests.cs

```diff
@@ -0,0 +1,504 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.XunitExtensions;
+using MongoDB.Driver.Core.Configuration;
+using MongoDB.Driver.Encryption;
+using MongoDB.Driver.TestHelpers;
+using MongoDB.Libmongocrypt;
+using Xunit;
+
+namespace MongoDB.Driver.Tests.Specifications.client_side_encryption.prose_tests
+{
+    // TODO: move into ClientEncryptionProseTests.cs
```

Comment:
should be moved into ClientEncryptionProseTests as soon as the spec will be ready

Reply:
Kevin just opened a PR on the specs repo with YAML/JSON based tests.  Can this be replaced with those?

Reply:
sure, will do it today